Fix indentation issue

cyxou opened the pull request:
This fixes #1

cyxou committed:
Set theme jekyll-theme-slate

cyxou committed:
Fix intendation for the config mount 'emptyDir'. Fixes #1

brandon099 reviewed (commented):
Are you intending to change the theme for the landing page at the hosted chart URL? If so, that's on a separate `gh-pages` branch.

brandon099 reviewed (changes requested):
Good catch -- I have persistence enabled in addition to not linting, so I never ran into that issue.
Also, could you bump the version in Chart.yaml?

If you could remove the theme commit for _config.yaml, and create a new PR against the `gh-pages` branch, I'll pull it in over there.

cyxou commented:
Oh, I apologize, that theme change was unintentional.
I've also found another issue with `externalTrafficPolicy` being set by default which produces an error like this:
```
Service "wg-easy-wg" is invalid: spec.externalTrafficPolicy: Invalid value:
    "Local": ExternalTrafficPolicy can only be set on NodePort and LoadBalancer service
```
I'll create another PR addressing all the issues I've found
